libnavigation-base/src/main/java/com/mapbox/navigation/base/route/RouterOrigin.kt

```diff
@@ -17,10 +17,24 @@ sealed class RouterOrigin {
     object Onboard : RouterOrigin()
 
     /**
-     * Can be used as a router origin of custom routes source, different from [Offboard] and [Onboard]. The
-     * SDK doesn't operate with [obj], it might be used to spread any additional data.
-     *
-     * @param obj is used to provide additional data with [Custom] router origin
+     * Can be used as a router origin of custom routes source, different from [Offboard] and [Onboard].
      */
-    data class Custom @JvmOverloads constructor(val obj: Any? = null) : RouterOrigin()
+    class Custom() : RouterOrigin() {
```

Comment:
NN doesn't have a chance to receive `Any?`(`@Nullable Object`) as the work with strict types. Suggesting here is deprecate `obj`, we don't receive any feedback somebody use it